pyodide-build/pyodide_build/create_xbuildenv.py

```diff
@@ -86,14 +91,24 @@ def copy_wasm_libs(xbuildenv_path: Path) -> None:
             shutil.copy(pyodide_root / path, xbuildenv_root / path)
 
 
-def main(args: argparse.Namespace) -> None:
-    pyodide_root = get_pyodide_root()
-    xbuildenv_path = pyodide_root / "xbuildenv"
+def create(
+    path: str | Path,
+    pyodide_root: Path | None = None,
+    *,
+    skip_missing_files: bool = False,
+) -> None:
+    if pyodide_root is None:
+        pyodide_root = get_pyodide_root()
+
+    xbuildenv_path = Path(path) / "xbuildenv"
```

Comment:
We should start putting the Pyodide version into these paths!

Reply:
Yes, indeed :) We also need to fix hardcoded xbuildenv paths.